core/src/main/java/org/elasticsearch/search/SearchService.java

```diff
@@ -546,6 +567,7 @@ final SearchContext createContext(ShardSearchRequest request, @Nullable Engine.S
                 keepAlive = request.scroll().keepAlive().millis();
             }
             context.keepAlive(keepAlive);
+            context.lowLevelCancellation(lowLevelCancellation);
```

Comment:
sorry I just realized this is the createContext method and not executeFetchPhase as I initially thought.

then I'm wondering if we could have the lowLevelCancellation only on DefaultSearchContext rather than the SearchContext base class?